File: app/src/main/java/com/app/applemvp/mvp/MainPresenter.java
package com.app.applemvp.mvp;

import com.base.applemvp.common.BasePresenter;

/**
 * applehsp  on 2018/5/17.
 */

public class MainPresenter extends BasePresenter<MainView> {


    public MainPresenter() {
    }


    /**
     * 初始化
     */
    public void appSync(String message){
        getView().appSync(message);
    }




}

File: app/src/main/java/com/app/applemvp/mvp/MainPresenter.kt
package com.app.applemvp.mvp

import android.util.Log
import com.base.applemvp.common.BasePresenter

/**
 * applehsp  on 2018/5/17.
 */
class MainPresenter : BasePresenter<MainView>() {
    /**
     * 初始化
     */
    fun appSync(message: String) {
        view!!.appSync("xxxxdddddd")
    }
}